docs/guides/authentication.md

```diff
@@ -26,12 +26,29 @@ To use the client, you need to initialize it with your API key:
 import MedrunnerApiClient from "@medrunner/api-client";
 
 const apiConfig = {
-  token: "YOUR_API_KEY",
+  refreshToken: "YOUR_API_TOKEN",
 };
 
 const api = MedrunnerApiClient.buildClient(apiConfig);
 ```
 
+### Exchange the refresh token for an access token
+
+The access token can be used to authenticate API calls when not using the TypeScript client. 
+
+```ts
+  import MedrunnerApiClient from "@medrunner/api-client";
+
+  const apiConfig = {
+    refreshToken: "YOUR_API_KEY",
+  };
+
+  const api = MedrunnerApiClient.buildClient(apiConfig);
+  const token = await api.staff.tokenManager.getAccessToken();
+
+  console.log(token); // YOUR_ACCESS_TOKEN
+```
+
```

Comment:
This shows how to get an access token, but only by using the typescript client. Access tokens are short-lived and refresh tokens must be exchanged to get a new one, so I'm concerned this might actually just confuse users into thinking they can just do this once to get a token that they can use forever, which is not the case.

Personally, I'd remove this. Any documentation for how to interact with the API outside of the client also really isn't within the scope of these docs to begin with 😄